akka-remote/src/main/scala/akka/remote/DefaultFailureDetectorRegistry.scala

```diff
@@ -47,6 +47,13 @@ class DefaultFailureDetectorRegistry[A](detectorFactory: () ⇒ FailureDetector)
               failureDetector.heartbeat()
             case None ⇒
               val newDetector: FailureDetector = detectorFactory()
+
+              // FIXME: fix this binary compatibility hack in 2.6
```

Comment:
We will probably keep 2.6 binary compatible also so there is not much point with this FIXME.
Change it to a comment that it was introduced as a `var` because of binary compatibility constraints.